### ISSUE TITLE
Support jshint/jslint special comment?

### ISSUE THREAD

**Issue by unc0** [ckknight/gorillascript#10]
``` coffeescript
/*global $:false, jQuery:false */
alert "Hello, GorillaScript!"
```

I tested in `gorilla -c -p`, compiler deletes that special comment.

**Comment by ckknight** [comment]
I was thinking about solving the global access issue by having an explicit statement at the top of a file such as

```
global jQuery // or something like global $ = jQuery or global $: jQuery for aliasing

jQuery #($)
  $("body").do-stuff()
```

Would turn into

```
let jQuery = GLOBAL.jQuery

jQuery #($)
  $("body").do-stuff()
```

At which point I can then test every ident access to assure that it is either in-scope or an expected primordial (e.g. String, setTimeout)

I think this would be a better solution, but I'd like to hear feedback.

**Comment by unc0** [comment]
Yes, I think that's better, no more grunt jshint tasks in my Gruntfile.
